src/libaktualizr/primary/aktualizr_test.cc

```diff
@@ -1163,6 +1220,7 @@ int main(int argc, char** argv) {
     std::cerr << "Error: " << argv[0] << " requires the path to the base directory of uptane repos.\n";
     return EXIT_FAILURE;
   }
+  std::cout << "MIKESUL:  repos" << argv[1] << std::endl;
```

Comment:
good catch, could'n figured out what the correct path to the repo so I can run & debug a specific test (t_aktualizr)

Reply:
What you were looking for is here I believe: https://github.com/advancedtelematic/aktualizr/blob/a4d3c4a44e5ea41721a1192d9cc6f02a83a03b72/src/libaktualizr/primary/CMakeLists.txt#L16

Or run the test with `-V`: `ctest -V '^test_aktualizr$'.

But in some cases, it's useful to inspect temporary directories. In that case, I use gdb breakpoints to pause it so that they are not deleted yet.